Description: Add more grub.d scripts to the list that should be run only once
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: no
Last-Update: 2013-12-23

Index: b/util/grub-mkconfig.in
===================================================================
--- a/util/grub-mkconfig.in
+++ b/util/grub-mkconfig.in
@@ -275,7 +275,9 @@
 for i in "${grub_mkconfig_dir}"/* ; do
   case "$i" in
       "${grub_mkconfig_dir}"/00_header \
+      | "${grub_mkconfig_dir}"/05_debian_theme \
       | "${grub_mkconfig_dir}"/30_os-prober \
+      | "${grub_mkconfig_dir}"/30_uefi-firmware \
       | "${grub_mkconfig_dir}"/40_custom \
       | "${grub_mkconfig_dir}"/41_custom)
 	      echo
